File: 06_second_highest/SecondHighest.java
import java.util.*;

public class SecondHighest{
    public static void main(String[] args){
        int arr[] = {8,5,2,6,7,9,12};
        Arrays.sort(arr);

        for(int i : arr){
            System.out.print(i + " ");
        }
        int n = arr.length;
        System.out.println("Highest: " + arr[n-1]);
        System.out.println("Second Highest: " + arr[n-2]);
    }
}

File: 06_second_highest/second_highest.py

arr = [8,5,2,6,7,14,9,12]

highest = arr[0]
second_highest = arr[0]

for i in arr:
    if i > highest:
        second_highest = highest
        highest = i
    elif i > second_highest:
        second_highest = i

print(f"Highest: {highest}")
print(f"Second Highest: {second_highest}")